src/lib/structures/Command.ts

```diff
@@ -48,6 +55,7 @@ export abstract class Command<T = Args> extends AliasPiece {
 		super(context, { ...options, name: (options.name ?? context.name).toLowerCase() });
 		this.description = options.description ?? '';
 		this.detailedDescription = options.detailedDescription ?? '';
+		this.category = options.category ?? this.path.split(sep).reverse()[1] ?? '';
```

Comment:
This default assumes very heavily that commands are within sub-folders, since otherwise they'd get wrongly assigned to `commands`.

Reply:
That being said, it's within my plans to make `@sapphire/pieces@v3` with relative-from-root-directory support, which I intend to use for calculating category in commands, and full route building in `plugin-api` Route.

Reply:
I have improved the method.